contributor/making-release.md

```diff
@@ -158,7 +204,7 @@ Once a collection is created for a major release, developers can make documentat
 
 **For 1.x minor & bugfix releases,** copy `accumulo_user_manual.html` generated for release to the `1.x/` directory in the [accumulo-website repo][website-repo].
 
-### Javadocs
+## Update Javadocs
 
 Javadocs are easy to update. Using the latest JDK7 or later (at least JDK 7u21
```

Comment:
We already require Java 8 to run the build, even for 1.9 branch, so this whole JDK7 stuff is irrelevant.